src/QMCWaveFunctions/BsplineFactory/SplineC2C.h

```diff
@@ -64,7 +64,7 @@ class SplineC2C : public BsplineSet
   std::shared_ptr<MultiBspline<ST>> SplineInst;
 
   ///Copy of original splines for orbital rotation
-  std::shared_ptr<std::vector<RealType>> coef_copy_;
+  std::shared_ptr<std::vector<ST>> coef_copy_;
```

Comment:
This is a valid change. coef_copy should keep the same precision as the original coefs.